collect_app/src/main/java/org/odk/collect/android/formentry/audit/AuditEventLogger.java

```diff
@@ -269,11 +262,26 @@ boolean isAuditEnabled() {
         return auditConfig != null;
     }
 
-    ArrayList<AuditEvent> getAuditEvents() {
-        return auditEvents;
-    }
-
     List<Location> getLocations() {
         return locations;
     }
+
+    public Boolean isUserRequired() {
```

Comment:
Use primitive `boolean`?

Reply:
This is my Kotlin self taking hold! I guess in Java there is a minor but definite performance hit though.

Reply:
For me it's not about the performance hit, it's about the problems nulls can cause. I find it especially bad with `Boolean` because they end up with three states which feels very wrong.